modules/rubiconBidAdapter.js

```diff
@@ -10,11 +10,9 @@ function isSecure() {
 }
 
 // use protocol relative urls for http or https
-const FASTLANE_ENDPOINT = '//fastlane.rubiconproject.com/a/api/fastlane.json';
-const VIDEO_ENDPOINT = '//fastlane-adv.rubiconproject.com/v1/auction/video';
-const SYNC_ENDPOINT = 'https://eus.rubiconproject.com/usync.html';
-
-const TIMEOUT_BUFFER = 500;
+export const FASTLANE_ENDPOINT = '//fastlane.rubiconproject.com/a/api/fastlane.json';
+export const VIDEO_ENDPOINT = '//prebid-server.rubiconproject.com/openrtb/auction';
```

Comment:
@dbridges12 any concerns?

Reply:
Endpoint updated with `2` added to `openrtb`